composer.json

```diff
@@ -5,16 +5,6 @@
     "keywords": ["php","html","form"],
     "homepage": "http://code.google.com/p/php-form-builder-class/",
     "license": "GNU GPL v3",
-    "authors": [
```

Comment:
hmm... this is not what I want to push to your repo )

I removed this section in my branch because of composer required field "name" in "authors" section.